src/pages/payroll/PayrollHeadPanel.js

```diff
@@ -181,6 +181,18 @@ class PayrollHeadPanel extends FormPanel {
               readOnly={readOnly}
             />
           </Grid>
+          <Grid item className={classes.item}>
+            <FormControlLabel
+              label={formatMessage(intl, 'payroll', 'payroll.includedUnpaid')}
+              control={(
+                <Checkbox
+                  checked={!!payroll && !!payroll.includedUnpaid && payroll.includedUnpaid}
```

Comment:
I think it can be shorten to: 
`
checked={!!payroll.includedUnpaid}
`

Reply:
ok changed and it works too